### ISSUE TITLE
Upgrade jQuery UI Dialog 1.11.4 to 1.12

### ISSUE THREAD

**Issue by dpatanker** [MISP/MISP#5190]
[feature upgrade]

Hi MISP Team,

A great platform and we are in process to move to production. One of the findings was usage of jquery ui dialog 1.11 API is deprecated and was wondering if there are plans to update this in future MISP upgrade?

Many Thanks

**Comment by iglocska** [comment]
Yeah, just had a look at where we actually use jquery-ui and only found one super obscure place so far. Think we can just remove it as a dependency and refactor the feature where we use it.
